docs/databricks.md

```diff
@@ -0,0 +1,120 @@
+$ wget https://search.maven.org/remotecontent?filepath=ai/rapids/xgboost4j-spark/0.90-Beta/xgboost4j-spark-0.90-Beta.jar
+``` 
+
+* Download Dataset: https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip
+
+```
+$ wget  https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip
+$ unzip mortgage.zip
+$ find . -type f -print|sort
+./data/mortgage/csv/test/mortgage_eval_merged.csv
+./data/mortgage/csv/train/mortgage_train_merged.csv
+``` 
+
+* Copy the data into Databricks DBFS:
+
+```
+Go to "Import and Explore Data" 
+Create New Table - default location is /FileStore/tables
+Select mortgage_eval_merged.csv and mortgage_train_merged.csv to upload.
+```
+
+* Upload the XGBoost-4j Spark jars
+
+```
+Go to "Import Library"
+Select "Upload" and then "Jar"
+Select the 3 XGBoost Spark jars and upload them. Save the file names and locations for the step below.
+```
+
+* Add a Startup init script  
+In a terminal create an init script that has commands to copy the jars you imported above over the existing Databricks provided XGBoost jars. Note you will have to change the jar names below to the ones you uploaded.  If you didn't save the names go to the Databricks file browser and look in /FileStore/jars/ If you are using a runtime other then Databricks 5.3 or 5.4 you will have to check the versions of the databricks provided jars in /databricks/jars and update the script accordingly.  
+
+```
+$ cat /dbfs/databricks/scripts/init.sh
+sudo cp /dbfs/FileStore/jars/2958ae11_357c_4f5d_b9ee_3212e3c0ec5c-xgboost4j_0_90_BETA-567a4.jar /databricks/jars/spark--maven-trees--ml--xgboost--ml.dmlc--xgboost4j--ml.dmlc__xgboost4j__0.81.jar
+sudo cp /dbfs/FileStore/jars/492544bd_53eb_42b7_a9f3_ba682c991839-cudf_0_8_BETA-64b01.jar /databricks/jars/
+sudo cp /dbfs/FileStore/jars/7b5344c1_38b1_4f9c_a951_f1bce67eb20b-xgboost4j_spark_0_90_BETA-c8c97.jar /databricks/jars/spark--maven-trees--ml--xgboost--ml.dmlc--xgboost4j-spark--ml.dmlc__xgboost4j-spark__0.81.jar
+
+Upload the init.sh script into /databricks/scripts/init.sh.  See https://docs.databricks.com/user-guide/clusters/init-scripts.html for more details about configuring cluster-scoped init script.
+```
+
+Start A Databricks Cluster
+--------------------------
+1. Create a Databricks cluster that meets the above prerequisites. Make sure to use one of the ML Databricks runtimes.
+2. Disable autoscaling.
+3. Choose the number of workers that matches the number of GPUs you want to use.
+4. Select a worker type that has a GPU for the worker.
+5. Update the cluster Configuration "Advanced Options" to include an "Init Scripts". Add your init.sh script you uploaded above: "dbfs:/databricks/scripts/init.sh"
+6. Optionally add other configurations.
+
+Import the GPU Mortgage Example Notebook
+---------------------------
+1. See https://docs.databricks.com/user-guide/notebooks/notebook-manage.html on how to import a notebook.
```

Comment:
Could these links be shortened?
- Read [Managing Notebooks](https://docs.databricks.com/user-guide/notebooks/notebook-manage.html)
- Import this notebook: [XGBoost4j-Spark on Databrics](/notebook/databricks/xgboost_notebook_mortgage-gpu_scala.scala)